### DISCUSSION TITLE
How can I deselect all tree elements with a function in react?

### DISCUSSION THREAD

**Discussion by danielzeroconsulting** [lukasbach/react-complex-tree#415] [Q&A]
I need help to know how to deselect all tree elements with a function in react

**Comment by lukasbach** [comment]
You can get a React ref (see [docs on that topic](https://rct.lukasbach.com/docs/guides/refs/)), then call [`treeRef.current.selectItems([])`](https://rct.lukasbach.com/docs/api/interfaces/TreeRef#selectitems) or [`treeEnvironmentRef.current.selectItems([], treeId)`](https://rct.lukasbach.com/docs/api/interfaces/TreeEnvironmentRef#selectitems).
